docs/src/pages/[platform]/connected-components/liveness/troubleshooting/troubleshooting-react.mdx

```diff
@@ -0,0 +1 @@
+{ /* Page is blank due to Table of Contents bug workaround.  DO NOT REMOVE. */ }
```

Comment:
I might have missed it, but I didn't find anywhere mentioned about the bug. Where is that from?

Reply:
Bug(Docs): Table of Contents does not update properly when navigating between pages https://github.com/aws-amplify/amplify-ui/issues/3969